### DISCUSSION TITLE
Indentation option in config.ini

### DISCUSSION THREAD

**Discussion by QuAzI** [jotaen/klog#318] [Feature ideas & discussions]
I would like to use 2 spaces by default. But currently there is no option to use it from klog commands (klog start, klog track). Only manually. I know that klog should support 2 spaces as I can see
```
Unexpected indentation: Please correct the indentation of this line. Indentation
    must be 2-4 spaces or one tab. You cannot mix different indentation styles
    within the same record.
```

**Comment by jotaen** [comment]
Yeah, I think it would make sense in the longer run to be able to specify whitespace preferences in `config.ini`. I’ll think about this.

In the meantime, there is one tip that might help to slightly alleviate the situation (though still not as convenient): klog always respects (takes over) the indentation style that it finds in a file. So for new files, you’d have to correct the indentation style once from 4 spaces to 2 spaces after creating the first entry. klog will then continue to use 2 spaces for all new entries in that file.
